keyboards/rot13labs/hackboard/info.json

```diff
@@ -0,0 +1,124 @@
+{
+    "keyboard_name": "HACKBOARD",
+    "manufacturer": "rot13labs",
+    "url": "https://rot13labs.com",
+    "maintainer": "c0ldbru",
+    "usb": {
+        "vid": "0xBEEF",
+        "pid": "0xCAFE",
+        "device_version": "1.0.0"
+    },
+    "matrix_pins": {
+        "cols": ["A0", "B0", "A1", "B1", "A2", "B2", "A3", "B3", "A4", "B4", "A5", "A6", "A7", "D6", "D5", "D1", "D0", "D7"],
+        "rows": ["C2", "C3", "C7", "C4", "C6", "C5"]
+    },
+    "diode_direction": "COL2ROW",
+    "processor": "atmega32a",
+    "bootloader": "usbasploader",
+    "layouts": {
```

Comment:
```suggestion
    "community_layouts": ["tkl_ansi"],
    "layouts": {
```

Reply:
@dunk2k I added this change but now its throwing an error - I don't know a ton about community layouts yet but is it possible this error is being caused by the fact that the hackboard layout doesn't match the traditional tkl-ansi layout perfectly? Theres an extra key between esc and F1 on the hackboard.

Reply:
After digging into this it does appear that thats the issue - the hackboard doesnt support the tkl_ansi community layout. reverting this chnage resolves the issue.